indico/modules/rb/client/js/common/rooms/RoomEditDetails.jsx

```diff
@@ -0,0 +1,48 @@
+import {Form, Header} from 'semantic-ui-react';
```

Comment:
missing file header

Reply:
I'd move all the editing stuff into an `edit` subdirectory

Reply:
> missing file header

Thanks.